config.md

```diff
@@ -265,6 +265,19 @@ Annotations are key-value maps.
 }
 ```
 
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
```

Comment:
Maybe:

* **`imagePath`** (string, optional) path to the root filesystem image used by VM-based runtimes.

Why do you currently require an absolute path?